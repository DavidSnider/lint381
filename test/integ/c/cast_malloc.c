foo();
bar(malloc(sizeof(int)));
int* baz = (int*) malloc(sizeof(int));
