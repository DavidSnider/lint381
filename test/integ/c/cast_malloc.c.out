cast_malloc.c:3:12: error: Don't cast the result of malloc
int* baz = (int*) malloc(sizeof(int));
           ^^^^^^
